### PR TITLE
Use "find(id)" instead of "find_by(id: id)"

### PR DESCRIPTION
**Pull request by iago-silva**
"find_by(id: id)" was used as good practice

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -843,7 +843,7 @@ render status: :forbidden
     User.find_by_first_name_and_last_name('Bruce', 'Wayne')
```
Comment (pirj) [inline README.md]: Do you think it makes sense to update the desction to "Favor the use of `find_by`/`find`"?

**Comment by eliotsykes** [comment]
Take care of the different behaviour these have.

`User.find(id)` will raise an error if there is no user with that id. `User.find_by(id: id)` will return `nil`.

To avoid confusion, it may be better to remove the `id` examples entirely and use a different attribute that isn't the primary key, e.g. `email`.

**Comment by st0ck** [comment]
I believe we can't just replace `find_by(id: id)` by `find(id)`. These methods have different results.
`find(id)` will fail if there is no such id, while `find_by(id: id)` will return `nil` in this case.

File: README.md (hunk 1 above)
Comment (iago-silva) [reply README.md]: It makes sense, but the below suggestion seems be more careful 🤠.

**Comment by iago-silva** [comment]
> Take care of the different behaviour these have.
> 
> `User.find(id)` will raise an error if there is no user with that id. `User.find_by(id: id)` will return `nil`.
> 
> To avoid confusion, it may be better to remove the `id` examples entirely and use a different attribute that isn't the primary key, e.g. `email`.

File: README.md [hunk 2]
```diff
@@ -843,7 +843,7 @@ render status: :forbidden
...
 
     # good
-    User.find(id)
+    User.find_by(email: email)
```
Comment (eliotsykes) [inline README.md]: The 2 corresponding `#bad` examples above should also use `email` instead of `id`.

**Comment by iago-silva** [comment]
Do I need do more something to merge it?

**Pull request merged**